src/table/head/table-head-expand.component.ts

```diff
@@ -1,15 +1,42 @@
 import {
 	Component,
-	HostBinding
+	EventEmitter,
+	HostBinding,
+	Input,
+	Output
 } from "@angular/core";
+import { I18n } from "carbon-components-angular/i18n";
+import { Observable } from "rxjs";
 
 @Component({
 	// tslint:disable-next-line: component-selector
 	selector: "[cdsTableHeadExpand], [ibmTableHeadExpand]",
 	template: `
-		<ng-content></ng-content>
+		<button
+			class="cds--table-expand__button"
+			[attr.aria-label]="getAriaLabel() | async"
+			(click)="change.emit(!allRowsExpanded)"
+		>
+			<svg cdsIcon="chevron--right" size="16" class="cds--table-expand__svg"></svg>
+		</button>
 	`
 })
 export class TableHeadExpand {
 	@HostBinding("class.cds--table-expand") hostClass = true;
+
+	@Input() allRowsExpanded = false;
+
+	@Output() change = new EventEmitter<boolean>();
```

Comment:
We can probably just call it `expanded` and `expandedChange` to allow for double binding. This is in case user is creating their own table.